care/facility/models/facility.py

```diff
@@ -50,6 +51,7 @@
 ROOM_TYPES.extend(BASE_ROOM_TYPES)
 
 REVERSE_ROOM_TYPES = reverse_choices(ROOM_TYPES)
+REVERSE_FEATURE_CHOICES = reverse_choices(FEATURE_CHOICES)
```

Comment:
we can use a new integer choice enum `FacilityFeature`
keep the old one in place, just replace it for the new field